docs/src/examples/components/Icon/States/IconExampleDisabled.shorthand.tsx

```diff
@@ -3,10 +3,10 @@ import { Icon } from '@stardust-ui/react'
 
 const IconExampleDisabled = () => (
   <div>
-    <Icon disabled name="users" size="big" />
-    <Icon disabled name="users" size="big" variables={{ color: 'red' }} />
-    <Icon disabled name="users" size="big" variables={{ color: 'orange' }} />
-    <Icon disabled name="users" size="big" variables={{ color: 'yellow' }} />
+    <Icon disabled name="umbrella" size="big" />
+    <Icon disabled name="umbrella" size="big" variables={{ color: 'blue' }} />
+    <Icon disabled name="umbrella" size="big" variables={{ color: 'red' }} />
+    <Icon disabled name="umbrella" size="big" variables={{ color: 'orange' }} />
```

Comment:
We should update this branch with the latest master and try to fixed the other components that use icons, as well as their examples, because currently those are not working.